src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlDocument.cs

```diff
@@ -61,52 +61,54 @@ public void WriteHash(HashAlgorithm hash, DocPosition docPos, AncestralNamespace
             }
         }
 
+#pragma warning disable CS8765 // Nullability doesn't match overriden member
         public override XmlElement CreateElement(string prefix, string localName, string namespaceURI)
+#pragma warning restore CS8765
```

Comment:
Why is this being suppressed rather than making `CreateElement` match the base (which presumably would mean making prefix and namespaceURI both nullable)?  If the issue is CanonicalXmlElement's constructor, I'd expect it should also have prefix and namespaceURI as nullable, and if the issue in doing that is XmlElement's protected internal ctor having prefix as non-nullable, that looks like a bug that should be fixed: I don't see anything preventing it from being nullable.

Reply:
That is exactly the problem. I will create a bug report and PR.